.github/workflows/04_release_deploy.yml

```diff
@@ -156,6 +174,7 @@ jobs:
     with:
       environment: ${{ matrix.environment }}
       target: ${{ needs.setup.outputs.environment }}
+      component: ${{ inputs.component }}
```

Comment:
this one ?

Reply:
it allows you to choose the deployment of fdr microservice or cron